treeherder/model/derived/jobs.py

```diff
@@ -2093,6 +2097,12 @@ def store_performance_series(
 
             db_series_json = performance_series[0]['blob']
 
```

Comment:
Yeah agree :-)

Annoyingly this was the pattern I used initially, but changed after seeing:
https://github.com/mozilla/treeherder/blob/57453c271501619903b7d444cce6e5acb8551178/treeherder/model/derived/artifacts.py#L90-L94